networkx/algorithms/minors.py

```diff
@@ -9,22 +9,65 @@
 from networkx.exception import NetworkXException
 from networkx.utils import arbitrary_element
 
-__all__ = ["contracted_edge", "contracted_nodes", "identified_nodes", "quotient_graph"]
+__all__ = [
+    "contracted_edge",
+    "contracted_nodes",
+    "equivalence_classes",
+    "identified_nodes",
+    "quotient_graph"
+]
 
 chaini = chain.from_iterable
 
 
 def equivalence_classes(iterable, relation):
-    """Returns the set of equivalence classes of the given `iterable` under
-    the specified equivalence relation.
+    """Returns equivalence classes of `iterable` given by `relation`.
```

Comment:
It is more correct to say that the equivalence classes are of the relation, not the iterable.

Reply:
Ahh... good... That's an important grammar that I somehow have missed.  Thanks!